internal/descriptor/descriptor.go

```diff
@@ -37,6 +50,16 @@ func ShortDigest(desc ocispec.Descriptor) (digestString string) {
 	return digestString
 }
 
+// PlainDescriptor returns a plain descriptor that contains only MediaType, Digest and
+// Size.
+func PlainDescriptor(desc ocispec.Descriptor) ocispec.Descriptor {
```

Comment:
nit: We can simply call this function `Plain` as it is in the package `descriptor`, so it can be used like `descriptor.Plain(desc)`.
BTW should we have a unit test for this?

Reply:
Renamed the function